packages/mds-transactions-service/package.json

```diff
@@ -19,7 +19,7 @@
     "test:eslint": "eslint --ignore-path ../../.gitignore '**/*.ts'",
     "test:unit": "DOTENV_CONFIG_PATH=../../.env jest",
     "ts-node": "DOTENV_CONFIG_PATH=../../.env node --experimental-repl-await -r ts-node/register -r dotenv/config",
-    "typeorm": "yarn ts-node ./node_modules/.bin/typeorm",
+    "typeorm": "yarn ts-node ../../node_modules/typeorm/cli.js",
```

Comment:
Unrelated to your work, but this is required to get typeorm commands to work on Windows.